packages/babel-parser/src/options.js

```diff
@@ -19,6 +19,7 @@ export type Options = {
   strictMode: ?boolean,
   ranges: boolean,
   tokens: boolean,
+  createParenthesizedExpressions: boolean,
```

Comment:
This also needs to be added to `babel-parser/typings/index.d.ts`

Reply:
I assume you mean [packages/babel-parser/typings/babel-parser.d.ts](https://github.com/babel/babel/blob/master/packages/babel-parser/typings/babel-parser.d.ts)